app/views/claims/_ineligibility_reason_ineligible_claim_school.html.erb

```diff
@@ -1,15 +1,22 @@
-<h1 class="govuk-heading-xl">
-  This school is not eligible
-</h1>
+<% if params[:tried_all_schools] %>
```

Comment:
I was actually thinking this would go in the main ineligible view, rather than in these two partials. So like this basically: https://github.com/DFE-Digital/dfe-teachers-payment-service/commit/f59a0c2461a9627db039dfd3e694a3143015b77b